### PR TITLE
[Package Importer] Draft 1.5 

### PR DESCRIPTION
**Pull request by jamesnw**
- Specify no throws for package importer misses

See https://github.com/sass/dart-sass/pull/2130/files/e04b10982423d44237cb4d976c460e59478d731d#r1458161774

### CODE REVIEW

File: proposal/package-importer.d.ts.md [hunk 1]
```diff
@@ -330,6 +330,10 @@ Package Importers must reject the following patterns:
 * A URL whose path begins with `/`.
 * A URL with non-empty/null username, password, host, port, query, or fragment.
 
+If the conventions or specifications for an environment disallow any other URL
+patterns, the Package Importer must not reject, and instead return `null`. This
```
Comment (nex3) [inline proposal/package-importer.d.ts.md]: "reject" isn't a well-defined term here. I'd say "must return `null` rather than throwing an error."

**Review by nex3 — CHANGES_REQUESTED** [review]
It's probably also a good idea to specify that `new NodePackageImporter()` throws an error if a path isn't passed and the default can't be loaded.

File: proposal/package-importer.d.ts.md (hunk 1 above)
Comment (jamesnw) [reply proposal/package-importer.d.ts.md]: Updated in https://github.com/sass/sass/pull/3777/commits/26d30d7b62448ee60074eb6e8d6a5cbbd727cb0d

**Comment by jamesnw** [comment]
> It's probably also a good idea to specify that `new NodePackageImporter()` throws an error if a path isn't passed and the default can't be loaded.

Added in 517b1e2c16e25d5b7a3728dbb4f525993564cd35

File: proposal/package-importer.changes.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## Draft 1.5
+
+* Specify that Package Importers must not reject URL patterns that other Package
+  Importers may be able to canonicalize.
+
+* Specify that the Node Package Importer throws if `entryPointPath` is not
+    passed and `require.main.filename` is not available.
```
Comment (nex3) [inline proposal/package-importer.changes.md]: Nit: fix indentation

**Review by nex3 — APPROVED** [review]

File: proposal/package-importer.changes.md (hunk 2 above)
Comment (jamesnw) [reply proposal/package-importer.changes.md]: Fixed in 3ac835a42cf5a7f691d7a92d613cbf81eaf29531

**Pull request merged**